contracts/exchange/test/utils/isolated_exchange_wrapper.ts

```diff
@@ -28,7 +28,7 @@ export interface IsolatedExchangeEvents {
     transferFromCalls: DispatchTransferFromCallArgs[];
 }
 
-export type Order = OrderWithoutDomain;
+export type Order = Order;
```

Comment:
This need to be re-exported like this anymore?

Reply:
It's a testing utility class. I figured it was good to keep the abstraction in place.